src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  width: 96px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
It's better to use a relative path to the image file instead of an absolute path. This way, your code will still work even if the location of the CSS file changes. Replace '/src/images/star.svg' with '../images/star.svg'. The same applies to the 'star-active.svg' image.